core/embed/rust/src/translations/translated_string.rs

```diff
@@ -10,10 +10,9 @@ impl TranslatedString {
             .unwrap_or(self.untranslated())
     }
 
-    pub fn map_translated<F, T>(self, fun: F) -> T
+    pub fn map_translated<'a, F, T>(self, fun: F) -> T
     where
-        F: for<'a> FnOnce(&'a str) -> T,
-        T: 'static,
+        F: FnOnce(&'a str) -> T,
```

Comment:
this change, and all changes downstream of this, are incorrect. please revert and use `tstring.map(|s| whatever_operation(s))` as originally intended

Reply:
Yes, noticed the discussion in the linked PRs. I'll monitor the development and port the changes accordingly.